lib/components/narrative/default/default-itinerary.js

```diff
@@ -131,6 +131,9 @@ const ITINERARY_ATTRIBUTES = [
     id: 'cost',
     order: 2,
     render: (itinerary, options, defaultFareKey = 'regular') => {
+      const fareInCents = getTotalFare(itinerary, options.configCosts, defaultFareKey)
+      const fare = fareInCents == null ? null : fareInCents / 100
+      if (fare == null) return null
```

Comment:
I was avoiding this condition in case there is fare information included that's intentionally 0 (free service). But I can rewrite as
`return fare !== null && ( 
   <Formatted Number 
    .... 
    /> 
 )
`

Reply:
Good point. It can stay as is.